pixano/pixano
# chore(front): add code linting and formating

ticket: https://github.com/pixano/pixano-project-manager/issues/2


**Description:** Addition of ESLint (for static code analysis) and Prettier (for code formatting).

**ESLint**: Basic configuration for Svelte and addition of three simple rules. We can consider adding more rules if desired!

**Prettier**: I added rules that seemed to be already applied. For projects within the 'apps' directory, I added two scripts: 'format' and 'lint' that help identify and address various issues.

**How to test** 
run `pnpm lint` or `pnpm format`  in `ui` folder.  
The above scripts can be used in CI. 

Please not that : 
-  for ease or PR reading, `pnpm format` has not been run.
- only "easy" TS and coding issues have been fixed. Some errors still occurs when run the lint scripts. Also, TS has been set to `strict: false` and lots of eslint basic rules have been disabled (see `eslintrc` for more details) causing massive tech debts. We should probably add that to the backlog? @jrabary @cpvannier
